shared/AppInsightsCore/src/JavaScriptSDK.Enums/TelemetryUpdateReason.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/**
+ * The TelemetryUpdateReason enumeration contains a set of bit-wise values that specify the reason for update request.
+ */
+ export const enum TelemetryUpdateReason {
+    /**
+     * Unknown.
+     */
+    Unknown = 0,
+
+    /**
+     * The configuration has ben updated or changed
+     */
+    //ConfigurationChanged = 0x01,
```

Comment:
Explicitly commenting out to ensure that I didn't include this portion of the code at this time.